awswrangler/data_api/redshift.py

```diff
@@ -35,8 +37,9 @@ class RedshiftDataApi(connector.DataApiConnector):
 
     def __init__(
         self,
-        cluster_id: str,
-        database: str,
+        cluster_id: str = "",
+        workgroup_name: str = "",
```

Comment:
My only concern here is that inserting `workgroup_name` between `cluster_id` and `database` would still lead to a breaking change for users who relied on positional arguments in their scripts. For instance, this code:
```
wr.data_api.redshift.connect(
    "aws-data-wrangler-1xn5lqxrdxrv3",
    "test_redshift",
    secret_arn="arn:aws:secretsmanager:us-east-1:111111111111:secret:aws-data-wrangler/redshift-ewn43d"
)
```
would break with this change because now the second argument is `workgroup_name` not `database` anymore.

Should we just put `workgroup_name` after `database`? This way the above is not broken and it's also more aligned with the `rds` code?

Reply:
good point, I've made the change.